ansible_collections/arista/avd/molecule/dhcp_provisioning/inventory/group_vars/DC1_FABRIC.yml

```diff
@@ -175,3 +185,67 @@ bfd_multihop:
   interval: 1200
   min_rx: 1200
   multiplier: 3
+
+# Custom node types
```

Comment:
Hmm given that the role takes only the first device in the template to derive the node_type_keys (yeah thats not great we agree) I don't think everything is being tested as we would expect here.
We probably need to create separate fabrics for the default behavior (that was tested before and is not now), the new syntax node_type_keys as you have here and the AVD 3.8 syntax node_type_keys.

I guess lets write 3 tests fabric / modify the playbook to run at three levels?

Reply:
I will work on the 3 fabrics test